### ISSUE TITLE
JetBrains: Unable to use Cody for Enterprise account.

### ISSUE THREAD

**Issue by jay-fibi** [sourcegraph/jetbrains#1464]
### Cody Version

v5.5.10-nightly

### IDE Information

IntelliJ IDEA 2023.3.4 (Community Edition)
Build #IC-233.14475.28, built on February 13, 2024
Runtime version: 17.0.10+1-b1087.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Registry:
  ide.experimental.ui=true
Non-Bundled Plugins:
  PythonCore (233.14475.9)
  com.soywiz.korge.korge-intellij-plugin (4.0.4)
  com.sourcegraph.jetbrains (5.5.10-nightly)
Kotlin: 233.14475.28-IJ

### Describe the bug

Platform: Windows

**Steps to reproduce:**

1. Have Cody plugin installed in your JB IntelliJ IDEA editor
2. Login to Cody with Enterprise user
3. Open Cody window, type something and hit enter or you can run "Explain Code" command

### Observed behavior:

"Cody encountered an error when processing your message" error message is displayed. In addition to this, the error says "ClientCodyIgnoreCompatibilityError: Cody for jetbrains version "5.5.10-nightly" doesn't match version constraint ">= 6.0.0""

### Expected behavior

Cody should respond to user question correctly.

### Additional context

**Screenshot:**

![Cody error](https://uploads.linear.app/15b96a4d-ffc0-481a-ba9c-9d840020f7bc/5050a6b0-1a0c-4328-9016-9229b01e1fca/9d1d864e-8c69-4923-9978-53e7056d8109?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzE1Yjk2YTRkLWZmYzAtNDgxYS1iYTljLTlkODQwMDIwZjdiYy81MDUwYTZiMC0xYTBjLTQzMjgtOTAxNi05MjI5YjAxZTFmY2EvOWQxZDg2NGUtOGM2OS00OTIzLTk5NzgtNTNlNzA1NmQ4MTA5IiwiaWF0IjoxNzE1NzYwMTAxLCJleHAiOjMzMjg2MzIwMTAxfQ.iRNDO2WRX0U6ZiHAy85nbIwhVPWlduC9OrLJT54F6U4)

**Comment by dominiccooney** [comment]
Slack thread with context: https://sourcegraph.slack.com/archives/C06Q9R098N4/p1715078795896099?thread_ts=1715038896.786949&cid=C06Q9R098N4

TL;DR:
- This is working as intended.
- Don't use sg02 to test enterprise features in 5.x product line.
- We will produce 6.x builds to test Cody Ignore, and you should use sg02 for that.

CC @taylorsperry, @kalanchan FYI

**Comment by kalanchan** [comment]
Closing this as we will address with a sourcegraph instance that doesn't have cody ignore enabled
